### PR TITLE
Create packagevariantset.yaml example

### PR DESCRIPTION
**Pull request by ganchandrasekaran**

### CODE REVIEW

**Comment by johnbelamaric** [comment]
We're getting rid of the v1alpha1 and going to the v1alpha2 implementation of PackageVariantSet, which is quite a bit more functional.

Take a look at #6 instead, it has 3 v1alpha2 PackageVariantSets. Also see https://github.com/GoogleContainerTools/kpt/pull/3930, in particular this file: https://github.com/GoogleContainerTools/kpt/pull/3930/files#diff-87293e2004e9b295a0880c583f473bf0a9ad6ba67e38d95d909e09ef7f7c0bc0

File: free5gc-smf-org-example/packagevariantset.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  targets:
+  - repositories:
+    - name: cluster-01
+      packageNames:
+      - foo-01
+      - foo-02
+      - foo-03
+    template:
+      downstream:
+        packageExpr: "packageName + '-' + repository.labels['env']"
```
Comment (johnbelamaric) [inline free5gc-smf-org-example/packagevariantset.yaml]: `packageName` is not correct, this should be either `target.package` or `packageDefault` (which have the same value in this case).

Comment (ganchandrasekaran) [reply free5gc-smf-org-example/packagevariantset.yaml]: @johnbelamaric Does it address your comment?

File: free5gc-smf-org-example/packagevariantset.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ spec:
       - foo-03
     template:
       downstream:
-        packageExpr: "packageName + '-' + repository.labels['env']"
+        packageExpr: "target.package"
```
Comment (johnbelamaric) [inline free5gc-smf-org-example/packagevariantset.yaml:33]: This works - I meant for it to be `"target.package + '-' repository.labels['env']"`, but for example purposes this is just fine too.

**Pull request merged**